templates/settings/admin.php

```diff
@@ -0,0 +1,37 @@
+<?php
+	script('facerecognition', 'admin');
+	style('facerecognition', 'facerecognition');
+?>
+
+<form id="facerecognition">
+	<div class="section">
+		<h2>
+			<?php p($l->t('Face Recognition'));?>
+			<a target="_blank" rel="noreferrer noopener" class="icon-info" title="<?php p($l->t('Open Documentation'));?>" href="https://github.com/matiasdelellis/facerecognition/wiki"></a>
+		</h2>
+		<div>
+			<h3>
+				<strong><?php p($l->t('Sensitivity'));?></strong>
+				<a target="_blank" rel="noreferrer noopener" class="icon-info" title="<?php p($l->t('Open Documentation'));?>" href="https://github.com/matiasdelellis/facerecognition/wiki/Sensitivity"></a>
+			</h3>
+			<p class="settings-hint"><?php p($l->t('The sensitivity determines how different the descriptors of the faces can be to consider them the same person.'));?></p>
```

Comment:
#consider

`The sensitivity determines how different faces need to be to consider them as the same person. Lower values will be more precise and will not put different people under same person, but it will tend to create more groups, even for same person. Higher values can see different persons under single same person, but it will create less number of clusters.`

or 
`
The sensitivity determines how different faces need to be to consider them as the same person. See <a href="">wiki</a> for details`

Reply:
I started writing a lot and then I preferred to leave it more concise by expanding on the wiki. The link is two lines of code above..

 * https://github.com/matiasdelellis/facerecognition/wiki/Sensitivity
 
> ...need to be..

The faces mo "need to be" different, at the most "can be" different.. 
.. but I agree to stop talking about the descriptors. :wink: